packages/augur-simplified/src/modules/market/trading-form.tsx

```diff
@@ -577,7 +575,11 @@ const TradingForm = ({
           updateInitialAmount={setAmount}
           initialAmount={''}
           maxValue={userBalance}
-          rate={!isNaN(breakdown?.ratePerCash) && `1 ${amm?.cash?.name} = ${isNaN(breakdown?.ratePerCash) ? '??' : breakdown?.ratePerCash} Shares`}
+          rate={
+            !isNaN(breakdown?.ratePerCash) ?
+              `1 ${amm?.cash?.name} = ${isNaN(breakdown?.ratePerCash) ? '??' : breakdown?.ratePerCash} Shares` :
+              `1 ${amm?.cash?.name} = ?? Shares`
```

Comment:
Not sure if this one should say: `Rate 1 ETH = ?? Shares` or `Rate 0 ETH = 0 Shares`